### PR TITLE
Joe Docs update

### PR DESCRIPTION
**Pull request by joehounsham**
-Adding new commercial faqs to faq section section
- Fixing anchor bug in Faqs
- Correcting readme to point to http://localhost:3000/docs
- Updated SIM7600E in favour of SIM7110E

### CODE REVIEW

File: pages/faq/troubleshooting/faq.md [hunk 1]
```diff
@@ -63,24 +68,19 @@ Starting from {{ $names.os.lower }} v2.0.7, the devices connect to the following
...
-Additionally, you should whitelist the following domains for the relevant ports above:
-* `*.{{ $names.dashboard_domain }}`
-* `*.docker.com`
-* `*.docker.io`
-
-
-Additionally, an outgoing connection to `mixpanel.com` is made. This is not a functional requirement for {{ $names.company.lower }}, but allows tracking of some useful metrics.
+Additionally, if the network your device is connecting to works with whitelisting, you should whitelist the following domains on port `80` and `443`:
+* `*.{{ $names.domain }}`
+* `*.pubnub.com`
```
Comment (shaunmulligan) [inline pages/faq/troubleshooting/faq.md]: @joehounsham why did you add back the pubnub requirement? We no longer use pubnub on any devices.

File: pages/faq/troubleshooting/faq.md [hunk 2]
```diff
@@ -63,24 +68,19 @@ Starting from {{ $names.os.lower }} v2.0.7, the devices connect to the following
...
-* `443` TCP - This is the most fundamental requirement - it is used to connect to the VPN and the web terminal, and many web endpoints using TLS (https://.)
+* `443` TCP - This is the most fundamental requirement - it is used to connect to the VPN and the web terminal, and many web endpoints using TLS (`https://`.)
 * `123` UDP - For NTP time synchronisation.
 * `53` UDP - For DNS name resolution.
 
 Each of these should work with outward only (and inward once outward connection established) firewall settings.
 
-Additionally, you should whitelist the following domains for the relevant ports above:
-* `*.{{ $names.dashboard_domain }}`
-* `*.docker.com`
```
Comment (shaunmulligan) [inline pages/faq/troubleshooting/faq.md]: Why did we remove docker.com domains? these are needed currently for host OS upgrades

File: templates/devices.html [hunk 3]
```diff
@@ -27,4 +27,9 @@ <h2>Currently supported</h2>
 			{% endif %}
 		{% endfor %}
 
+<<<<<<< HEAD
```
Comment (shaunmulligan) [inline templates/devices.html]: @joehounsham I assume these were left over when trying to sort out merge conflicts?

File: pages/faq/troubleshooting/faq.md (hunk 1 above)
Comment (joehounsham) [reply pages/faq/troubleshooting/faq.md]: Hmmm @shaunmulligan, I didn' touch this...

File: pages/faq/troubleshooting/faq.md (hunk 2 above)
Comment (joehounsham) [reply pages/faq/troubleshooting/faq.md]: Or this...

File: templates/devices.html (hunk 3 above)
Comment (joehounsham) [reply templates/devices.html]: The change here was modifying the contact email to solutions@balena.io

File: pages/faq/troubleshooting/faq.md [hunk 4]
```diff
@@ -141,11 +141,11 @@ Generally, we try to follow good OPSEC practices for our systems. We support 2FA
...
 
 ##### I have a device that is not on the supported devices list. Can it run on {{ $names.company.lower }}?
 
-There are a few options for devices that do not have an official device type on {{ $names.company.lower }}. If your device has an x86 architecture, you can try the [Intel NUC][nuc] image, which is built to support generic x86 devices. For other devices, you can [build your own][build-your-own] version of {{ $names.os.lower }} using our [open source repos][balenaOS]. To discuss custom board support, please contact sales@{{ $names.email_domain }}.
+There are a few options for devices that do not have an official device type on {{ $names.company.lower }}. If your device has an x86 architecture, you can try the [Intel NUC][nuc] image, which is built to support generic x86 devices. For other devices, you can [build your own][build-your-own] version of {{ $names.os.lower }} using our [open source repos][balenaOS]. To discuss custom board support, please contact sales@{{ $names.domain }}.
```
Comment (chrisys) [inline pages/faq/troubleshooting/faq.md]: This should remain as email_domain

**Comment by chrisys** [comment]
@joehounsham some weird regressions here - almost like your branch wasn't fully up to date or something?

**Comment by joehounsham** [comment]
> @joehounsham some weird regressions here - almost like your branch wasn't fully up to date or something?

Hmm, I forked the master repo yesterday....

**Comment by lurch** [comment]
Could this be another situation where the repo contains checked in versions of both the "source" and "output" files, and somebody has accidentally edited and checked in the "output" files"; when what they should have done is edited the "source" files, run the build process, and then checked in both the "source" and "output" files?

File: templates/devices.html (hunk 3 above)
Comment (shaunmulligan) [reply templates/devices.html]: @joehounsham I was more refering to the `<<<<<<< HEAD` which I assume you didn't plan on including?

File: pages/faq/troubleshooting/faq.md (hunk 1 above)
Comment (shaunmulligan) [reply pages/faq/troubleshooting/faq.md]: @joehounsham can you try revert these changes, we need to keep the networking section as it is.

**Comment by joehounsham** [comment]
@chrisys @shaunmulligan What do you reckon I should do here, do a fresh pull and start from scratch?

**Comment by shaunmulligan** [comment]
@joehounsham yeah that might be the best way forward, alternatively if you just point me to the FAQs that you want to add and I will PR them tomorrow for you? After we have them in, we can sit down with the solutions team and make sure you guys have a good way of making changes/PRs

**Comment by joehounsham** [comment]
> @joehounsham yeah that might be the best way forward, alternatively if you just point me to the FAQs that you want to add and I will PR them tomorrow for you? After we have them in, we can sit down with the solutions team and make sure you guys have a good way of making changes/PRs

It would be good for me to make the changes again I think, as the last PR I did was fine so It would be good to understand what has happened here for future ref. Shall i cancel this PR and start a fresh?

**Comment by shaunmulligan** [comment]
@joehounsham yup that would be good. Also one thing to remember is that each of you commits need to have the following format:
```
Description of the change

Change-type: patch
Signed-off-by: Shaun Mulligan <shaun@balena.io>

**Comment by sradevski** [comment]
@joehounsham I didn't notice your PR and I went ahead and fixed the links [here](https://github.com/balena-io/docs/pull/1052), so maybe you can base your new PR on top of it once that gets merged? Cheers.

**Comment by xginn8** [comment]
@joehounsham we're making a push to clean up PRs in the docs, are you still working on this? If not, can we close it for now?

**Comment by xginn8** [comment]
closing for now
